### PR TITLE
GL Polls [OGN] + [OUSD]

### PR DESCRIPTION
**Pull request by Gala-G**
Greenlight Polls for [OGN] and [OUSD]

### CODE REVIEW

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - [OUSD] - January 10, 2022
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:2]: Template is "$asset_code ($asset_name)". Remember, everything with a $ sign is something you need to change. So this should be "Community Greenlight Poll - OUSD (Origin Dollar) - January 10, 2022", same in the file name

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+summary: Signal your support or opposition to prioritising onboarding [OGN / OUSD].
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:3]: $asset_code ($asset_name)

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-01-10T16:00:00
+end_date: 2022-01-24T16:00:00
+---
+# Poll: Community Greenlight Poll - [OUSD] - January 10, 2022
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:16]: $asset_code ($asset_name)

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on January 10, 2022 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset [OUSD] should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:27]: $asset_code ($asset_name)

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Please review the collateral onboarding application [[OUSD] MIP6 Collateral Onboarding Application - Origin Protocol & Origin Dollar](https://forum.makerdao.com/t/12205) to inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* The 'greenlight score' will be calculated for [OUSD] using the formula `Score = 'Yes' Votes - 'No' Votes`
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:34]: $asset_code ($asset_name)

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+* The 'greenlight score' for [OUSD] will be added to the [Collateral Prioritisation Sheet](https://docs.google.com/spreadsheets/d/1IX9e2fyfz7djtDMKn5gMyGsyFxHoY75GncMbAjnSXrM/edit#gid=0) by the Governance Facilitators.
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:35]: $asset_code ($asset_name)

File: governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset [OUSD] should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
+
+Please review the collateral onboarding application [[OUSD] MIP6 Collateral Onboarding Application - Origin Protocol & Origin Dollar](https://forum.makerdao.com/t/12205) to inform your position before voting.
```
Comment (mkrorbrkr) [inline governance/polls/ MIP9 Community Greenlight Poll - [OUSD] - January 10, 2022.md:29]: You just need "thread" in the square brackets here, not the thread title

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
Hi Gala, seems to be the same thing repeated multiple times rather than multiple issues, which is a good thing! Hopefully won't take long to fix.

The file names both need update too but I can't comment on that

**Comment by mkrorbrkr** [comment]
Looking good @Gala-G, just need to change the file names and then I think we're good to go!

**Review by mkrorbrkr — APPROVED** [review]
LGTM! @prose11

**Review by prose11 — APPROVED** [review]
Nice work!

**Pull request merged**